cpp/tests/groupby/structs_tests.cpp

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright (c) 2019-2021, NVIDIA CORPORATION.
```

Comment:
Is this file entirely new? If so:
```suggestion
 * Copyright (c) 2021, NVIDIA CORPORATION.
```

Reply:
Yikes. Corrected.